src/Layer/C3DTilesLayer.js

```diff
@@ -311,6 +61,10 @@ class C3DTilesLayer extends GeometryLayer {
      * removed from the scene.
      * @param {C3DTExtensions} [config.registeredExtensions] 3D Tiles extensions managers registered for this tileset.
      * @param {Style} [config.style=null] - style used for this layer.
+     * @param {Boolean|Material} [config.overrideMaterials='false'] option to override the materials of all the layer's
```

Comment:
Can you remove the doc about this one so we don't encourage user to use it since it will be removed in the next major version? (deprecation message)